src/libstd/sync/mutex.rs

```diff
@@ -231,7 +231,7 @@ impl<T: ?Sized> Mutex<T> {
     #[stable(feature = "rust1", since = "1.0.0")]
     pub fn try_lock(&self) -> TryLockResult<MutexGuard<T>> {
         if unsafe { self.inner.lock.try_lock() } {
-            Ok(try!(MutexGuard::new(&*self.inner, &self.data)))
+            Ok(try!(unsafe { MutexGuard::new(&*self.inner, &self.data) }))
```

Comment:
With this new proliferation of `unsafe` blocks, can blocks like this just be merged with the one above? (e.g. have one surrounding block)